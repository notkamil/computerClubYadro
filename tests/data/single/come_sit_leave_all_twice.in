3
08:00 20:00
10
09:00 1 client1
10:00 2 client1 1
12:00 4 client1
14:00 1 client1
15:00 2 client1 1
17:00 4 client1