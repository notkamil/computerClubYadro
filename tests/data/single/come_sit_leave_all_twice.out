08:00
09:00 1 client1
10:00 2 client1 1
12:00 4 client1
14:00 1 client1
15:00 2 client1 1
17:00 4 client1
20:00
1 40 04:00
2 0 00:00
3 0 00:00